cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1469,7 +1469,7 @@ private void tryConnection(final ModelControllerClient client, ControllerAddress
             builder.addProperty(Util.NAME, Util.NAME);
 
             final long start = System.currentTimeMillis();
-            final long timeoutMillis = config.getConnectionTimeout() + 1000;
+            final long timeoutMillis = config.getConnectionTimeout() + 1000L;
```

Comment:
To overflow the timeouts adding 1000 to the `config.getConnectionTimeout()` integer value, the user should configure an unreal timeout value